diesel_cli/src/cli.rs

```diff
@@ -18,11 +18,37 @@ pub fn build_cli() -> App<'static, 'static> {
         .setting(AppSettings::VersionlessSubcommands)
         .arg(migration_dir_arg())
         .subcommand(SubCommand::with_name("run").about("Runs all pending migrations"))
-        .subcommand(SubCommand::with_name("revert").about("Reverts the latest run migration"))
-        .subcommand(SubCommand::with_name("redo").about(
-            "Reverts and re-runs the latest migration. Useful \
-             for testing that a migration can in fact be reverted.",
-        ))
+        .subcommand(
+            SubCommand::with_name("revert")
+                .about("Reverts the latest run migration")
+                .arg(
+                    Arg::with_name("REVERT_ALL")
```

Comment:
Why is this in SCREAMING_SNAKE_CASE? `Arg::with_name("all")` sounds better. Args are unique to only the SubCommands. So, you don't need `REVERT_` here. Similarly below.

Reply:
Ok thank you, I will make the changes.

Reply:
@pksunkara in fact I followed the naming convention previously used in this file. For example : 

```rust
.subcommand(
    SubCommand::with_name("generate")
        .about(
            "Generate a new migration with the given name, and \
             the current timestamp as the version",
        )
        .arg(
            Arg::with_name("MIGRATION_NAME")
                .help("The name of the migration to create")
                .required(true),
        )
        .arg(
            Arg::with_name("MIGRATION_VERSION")
                .long("version")
                .help(
                    "The version number to use when generating the migration. \
                     Defaults to the current timestamp, which should suffice \
                     for most use cases.",
                )
                .takes_value(true),
        )
```

To answer to your question.

Reply:
Yeah, I would say so. I am maintainer of clap so I know how people use it. But @weiznich might have some preferences here.

Reply:
I would prefer to just keep using the old style. Generally we try to avoid bigger cosmetic code refactoring, as this will "destroy" the history of that code. (It makes it harder to use something like `git blame` to get information about when the code was last changed, which allows to see the reason in the corresponding commit message)